app/src/main/java/com/myapps/toualbiamine/basketbusiness/Model/User.java

```diff
@@ -0,0 +1,38 @@
+package com.myapps.toualbiamine.basketbusiness.Model;
+
+
+public class User {
+        private String name;
+        private String password;
+        private String email;
+        private boolean isStaff;
+
+    public User() {
+
+}
+
+    public User(String email, String name, String password, Boolean isStaff) {
+        this.email = email;
+        this.name = name;
+        this.password = password;
+        this.isStaff = isStaff;
+}
+
+    public String getEmail() { return email; }
+
+    public void setEmail(String email) { this.email = email; }
+
+    public String getName() { return name; }
+
+    public void setName(String name) { this.name = name; }
+
+    public String getPassword() { return password; }
+
+    public void setPassword(String Password) { this.password = password; }
+
+    public boolean getIsStaff() {return isStaff; }
+
+    public void setIsStaff() { this.isStaff = isStaff; }
```

Comment:
Following the pattern for set functions, we need a parameter in the parenthesis. 
We are saying `The isStaff of THIS user is going to be isStaff`. However, there's no isStaff anywhere in the function. Follow the example above with `setName(..)` if needed!